lib/carbon/client.py

```diff
@@ -413,7 +413,7 @@ class CarbonLineClientProtocol(CarbonClientProtocol, LineOnlyReceiver):
 
   def _sendDatapointsNow(self, datapoints):
     for metric, datapoint in datapoints:
-        self.sendLine("%s %s %d" % (metric, datapoint[1], datapoint[0]))
+        self.sendLine("%s %f %d" % (metric, datapoint[1], datapoint[0]))
```

Comment:
Hmmm.... Would it convert all values to float explicitly?

Reply:
I think all values are casted to float anyway in the code